### DISCUSSION TITLE
Bank tags for specific

### DISCUSSION THREAD

**Discussion by MathieuLussier** [runelite/runelite#16530] [Ideas]
Is it possible to have a bank tag for a specific account? for example for cox i have a gear setup my main account and one for my iron. Instead of havind 2 bank tags. i would like it to show the bank tag only for certain specific accounts. Obviously some bank tags can be shared. Thank you

**Comment by Nightfirecat** [comment]
There's no way to share tags between accounts. You could use separate profiles to have different sets of tags and use those separate profiles on different accounts.
